docs/source/1.0/spec/aws/aws-auth.rst

```diff
@@ -148,6 +148,33 @@ literal string ``UNSIGNED-PAYLOAD`` is used when constructing a
 .. smithy-trait:: aws.auth#cognitoUserPools
 .. _aws.auth#cognitoUserPools-trait:
 
+------------------------
+``presignable`` trait
+------------------------
+
+Trait Summary
+    A presigner is a client-side utility that generates a presigned request for a given
+    operation invocation that may be used in contexts where direct calls may be
+    difficult or impossible.  This trait indicates that a client presigner could be
+    generated for the given operation.
+
+Trait selector
+    ``operation``
+Trait value
+    Annotation trait.
+
+The following example defines a service operation for which presigned requests can be generated for.
```

Comment:
You should include a directions on how to pre-sign a request, or at least link to any existing AWS docs that do the same.

Reply:
fixed

Reply:
fixed in https://github.com/awslabs/smithy/pull/1077